### PR TITLE
Feature/new blogposts

### PR DESCRIPTION
**Pull request by FranFin**
Added several new blogposts.
Addressed changes suggested by Ralph in PR #261.

### CODE REVIEW

File: _posts/2020-11-12-ROS_world.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: micro-ROS @ ROS World 2020! 
+author: francesca-finocchiaro
+---
+
+This year's ROS-Con, due to the pandemic, has been transformed into a virtual event, full of awesome presentations and discussions, the [ROS World 2020 Conference](https://roscon.ros.org/world/2020/)!
```
Comment (ralph-lange) [inline _posts/2020-11-12-ROS_world.md:6]:
```suggestion
This year's ROSCon, due to the pandemic, has been transformed into a virtual event, full of awesome presentations and discussions, the [ROS World 2020 Conference](https://roscon.ros.org/world/2020/)!
```

File: _posts/2020-12-16-ROS-In.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+In the second part of the talk, Francesca Finocchiaro provided an overview of the micro-ROS project with a focus on its latest features and developments, and subsequently Pablo Garrido Sanchéz presented an entry-level hands-on demonstration intended for people willing to get started with micro-ROS.
+
+<!-- Within this framework, we have delivered a talk in which we have provided an overview on the micro-ROS project with a special focus on its latest developments (Francesca Finocchiaro), and then presented an entry-level hands-on demonstration of its usage, intended for people willing to get started with micro-ROS (Pablo Garrido Sanchez).-->
+
+Find the presentations [here](/download/ROS-Industrial_workshop_ROS2.pdf) and [here](/download/ROSIn2020-FrancescaFinocchiaro.pdf).
```
Comment (ralph-lange) [inline _posts/2020-12-16-ROS-In.md:14]:
```suggestion
Find the presentations [here](/download/2020-12-16_Advanced_Execution_Management_with_ROS_2.pdf) and [here](/download/ROSIn2020-FrancescaFinocchiaro.pdf).
```

File: _posts/2020-12-16-ROS-In.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+<embed src="/download/ROS-Industrial_workshop_ROS2.pdf" type="application/pdf" width="100%" height="450px"/>
```
Comment (ralph-lange) [inline _posts/2020-12-16-ROS-In.md:16]:
```suggestion
<embed src="/download/2020-12-16_Advanced_Execution_Management_with_ROS_2.pdf" type="application/pdf" width="100%" height="450px"/>
```

File: _posts/2020-12-16-ROS-In.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+title: micro-ROS @ ROS-Industrial Conference 2020! 
+author: francesca-finocchiaro
+---
+
+micro-ROS was presented at the 2020 ROS-Industrial Conference in a joint talk given by Ralph Lange from Bosch, and by Francesca Finocchiaro and Pablo Garrido Sanchez from eProsima.
+
+In the first part of the talk, Ralph Lange discussed real-time support and determinism in both ROS 2 and micro-ROS, which is an important requirement for industrial automation. The talk also addressed the most recent developments added in the Foxy release for the Callback-group-level Executor and the roadmap for the Galactic release. Finally, he presented details on the rclc Executor for deeply embedded applications such as those addressed by micro-ROS.
```
Comment (ralph-lange) [inline _posts/2020-12-16-ROS-In.md:8]:
```suggestion
In the first part of the talk, Ralph Lange presented the ROS 2 Executor design and discussed latest developments towards real-time support and determinism, which are important requirements for industrial applications. The talk also addressed the most recent features added in the Foxy release and for the Galactic release. Finally, he presented details on the rclc Executor for deeply embedded applications such as those addressed by micro-ROS.
```

**Review by ralph-lange — CHANGES_REQUESTED** [review]
Very nice work, @FranFin!

File: _posts/2020-10-05-moveit.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+The pose data is then forwarded to the ROS world, where it is consumed by both Rviz and MoveIt. The former uses it directly to represent the position and orientation of the board in its graphical interface, while MoveIt uses it to calculate the movement that the virtual arm has to perform to "touch" it, according to its kinematic algorithms for motion planning. The resulting movement is then integrated into Rviz and represented by means of its [virtual *panda* robotic arm](https://github.com/ros-planning/panda_moveit_config), a standard tool employed by MoveIt in tutorials and graphic interfaces.
```
Comment (ralph-lange) [inline _posts/2020-10-05-moveit.md:13]:
```suggestion
The pose data is then forwarded to the ROS world, where it is consumed by both Rviz and MoveIt. The former uses it directly to represent the position and orientation of the board in its graphical interface, while MoveIt uses it to calculate the movement that the virtual arm has to perform to "touch" it, according to its kinematic algorithms for motion planning. The resulting movement is then integrated into Rviz and represented by means of its [virtual Panda robotic arm](https://github.com/ros-planning/panda_moveit_config), a standard tool employed by MoveIt in tutorials and graphic interfaces.
```

File: _posts/2020-11-18-luxonis.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: micro-ROS goes AIoT!
+author: francesca-finocchiaro
+---
+
+We’re working with the [LUXonis]() team on the integration of micro-ROS with an exciting AIoT (Artificial Intelligence of Things) project, involving the [DepthAI BW1092 board](https://shop.luxonis.com/products/bw1092). This small device enables interacting with LUXonis [DepthAI](https://luxonis.com/depthai) SoM from an [ESP32](https://www.espressif.com/en/products/socs/esp32) microcontroller.
```
Comment (ralph-lange) [inline _posts/2020-11-18-luxonis.md:6]:
```suggestion
We’re working with the [LUXonis](https://luxonis.com/) team on the integration of micro-ROS with an exciting AIoT (Artificial Intelligence of Things) project, involving the [DepthAI BW1092 board](https://shop.luxonis.com/products/bw1092). This small device enables interacting with LUXonis [DepthAI](https://luxonis.com/depthai) SoM from an [ESP32](https://www.espressif.com/en/products/socs/esp32) microcontroller.
```

File: _posts/2020-11-20-IS.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+In the first dataspace we have a ROS 2 publisher sending data over a Twist type message. More in detail, a *joy node* is in charge of obtaining displacement commands from a gamepad, which are then converted into ROS 2-understandable Twist type messages by a second *teleop_twist_joy node*.
+
+The second dataspace handles two subscriptors which receive the Twist topic sent by the *teleop_twist_joy node*. The first subscriptor is a Turtlesim application, in charge of displacing the classic virtual turtle in its graphical interface according to the indications received by the remote node. The second subscriptor is a micro-ROS Client, communicated with the ROS 2 datapace via an Agent which allows it to subscribe to the Twist topic. With the data received, micro-ROS is in charge of instructing a Kobuki Turtlebot2, operated by an [ESP32](https://www.espressif.com/en/products/socs/esp32) MCU running [FreeRTOS](https://www.freertos.org/index.html), to move accordingly.
```
Comment (ralph-lange) [inline _posts/2020-11-20-IS.md:10]:
```suggestion
The second dataspace handles two subscriptors which receive the Twist topic sent by the *teleop_twist_joy node*. The first subscriptor is a Turtlesim application, in charge of displacing the classic virtual turtle in its graphical interface according to the indications received by the remote node. The second subscriptions is a micro-ROS Client, communicated with the ROS 2 dataspace via an Agent which allows it to subscribe to the Twist topic. With the data received, micro-ROS is in charge of instructing a Kobuki Turtlebot2, operated by an [ESP32](https://www.espressif.com/en/products/socs/esp32) MCU running [FreeRTOS](https://www.freertos.org/index.html), to move accordingly.
```

**Comment by FranFin** [comment]
Thanks, @ralph-lange. Just addressed your commentaries!

**Comment by FranFin** [comment]
Tests are failing due to Twitter links that appear as broken but really are not.

**Review by jamoralp — APPROVED** [review]

**Pull request merged**